app/models/manageiq/providers/vmware/cloud_manager/refresh_parser.rb

```diff
@@ -58,6 +65,26 @@ def get_vms
     process_collection(@inv[:vms], :vms) { |vm| parse_vm(vm) }
   end
 
+  def get_vapp_templates
+    @inv[:orgs].each do |org|
+      org.catalogs.each do |catalog|
+        next if catalog.is_published && !@options.get_public_images || !catalog.is_published && !@options.get_private_images
```

Comment:
If you think turning off private catalogs doesn't make sense, I can remove it from this PR and only leave the setting for public ones.